cypress/integration/pages/onDemandRadio/tests.js

```diff
@@ -28,8 +28,16 @@ export default ({ service, pageType, variant, isAmp }) => {
         cy.request(
           `${Cypress.env('currentPath')}.json${dataEndpointOverride()}`,
         ).then(({ body: jsonData }) => {
-          if (!isAvailable(jsonData)) {
-            return cy.log(`Episode unavailable: ${Cypress.env('currentPath')}`);
+          const episodeAvailability = isAvailable(jsonData);
+          if (episodeAvailability !== 'available') {
+            if (episodeAvailability === 'future') {
+              throw new Error('Episode is in the future');
```

Comment:
I guess this is a good way of catching this problem in future... except it will only fail when the tests run. There is a risk that we could have a 6 hour period where we don't detect this.

Reply:
Yeah :( We will need to run it ourselves when we know Ares are making a change like they did yesterday

Reply:
If there is at least 1 on demand radio / TV marked as a smoke test, then these tests will run every time we do a Simorgh release. So it would actually block the deployment, if there was an unexpected AReS change. Is this what we want?

Reply:
I think we do want this, because in the case of the issue yesterday, there was a good period of time where users could not access the most recent episode on brand pages and we definitely want to stop that happening.